5calls/app/src/main/java/org/a5calls/android/a5calls/controller/StatsActivity.java

```diff
@@ -54,16 +68,16 @@
 public class StatsActivity extends AppCompatActivity {
     private static final String TAG = "StatsActivity";
     private static final int NUM_CONTACTS_TO_SHOW = 3;
-
+    private static final SimpleDateFormat usDateFormat = new SimpleDateFormat("MM/dd/yy", Locale.US);
```

Comment:
Can you store the date format in strings.xml? That way it can be localized depending on language.